package.json

```diff
@@ -17,7 +17,9 @@
     "react-router": "1.0.0"
   },
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "build": "./node_modules/.bin/webpack --progress --watch",
+    "start": "./node_modules/.bin/webpack; python manage.py runserver"
```

Comment:
How about this: we get rid of the start command, and developers can `npm build` and `python manage.py runserver` in separate tabs. It would be cool to do it both in one command, but this would be a good start for now.